Makefile

```diff
@@ -114,37 +122,42 @@ else
 endif
 
 $(K_JAR):
-	cd $(K_SUBMODULE) && mvn package -DskipTests -U -Dproject.build.type=$(K_BUILD_TYPE) $(K_MVN_ARGS)
+	cd $(K_SUBMODULE)                                                                                                                                                             \
+	    && mvn --batch-mode package -DskipTests -Dllvm.backend.prefix=/kframework -Dllvm.backend.destdir=$(CURDIR)/$(KEVM_LIB) -Dproject.build.type=$(K_BUILD_TYPE) $(K_MVN_ARGS) \
```

Comment:
Well, it's being installed to `$(CURDIR)/.build/lib/kevm/kframework`, so that the entire K install lives at `/usr/lib/kevm/kframework` on the target machine eventually at `make install`.

I'll try it with `prefix=$(CURDIR)/$(KEVM_LIB)/kframework`, and `destdir=/` I guess.

Reply:
I mean, that's not really a good idea. Certain paths get baked into the binaries we generate when we build the llvm backend, so you really want to specify the prefix that will actually exist at installation time. Which in this case would be `/usr/lib/kevm/kframework`.